apps/meet/src/components/SelectPersonView.tsx

```diff
@@ -16,6 +17,7 @@ export type SelectPersonViewProps = {
 };
 
 const SelectPersonView: React.FC<SelectPersonViewProps> = ({ page: { groupId }, setPage }) => {
+  const currentTimeMs = useCurrentTimeMs();
```

Comment:
Just to make sure I'm 100% understanding - this will force this component to re-render every 30s, right? Contrasting to this before where using `Date.now()` would only cause one render?

Reply:
Yep that's right 👍